spec/lib/tasks/clear_invalidated_earnings_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+require "rake"
+
+RSpec.describe "clear_invalidated_earnings" do
+  describe ":clear_invalidated_earnings", type: :task do
+    subject(:task) { Rake::Task["clear_invalidated_earnings"] }
+
+    before do
+      Rake.application.rake_require("tasks/clear_invalidated_earnings")
+      Rake::Task.define_task(:environment)
+      task.reenable
+      FormHandler.instance.use_real_forms!
+    end
+
+    context "when the rake task is run" do
+      context "and there are 2023 logs with invalid earnings" do
```

Comment:
it definitely will pass, but I wouldn't hate there being a test ensuring 2022 logs/valid 2023 logs remain unchanged and valid.

Reply:
Yep! Will add that!